shared/js/media/remote_controls.js

```diff
@@ -387,21 +355,19 @@ MediaRemoteControls.prototype.notifyMetadataChanged = function(metadata) {
  * @param {Object} status
  */
 MediaRemoteControls.prototype.notifyStatusChanged = function(status) {
-  // Send the new status via bluetooth.
-  if (this.defaultAdapter) {
-    // Don't send the interrupted statuses to the remote client because
-    // they are not the AVRCP statuses.
-    if (status === 'mozinterruptbegin') {
-      return;
-    }
-
-    var request = this.defaultAdapter.sendMediaPlayStatus(status);
-
-    request.onerror = function() {
-      console.log('Sending Playstatus error');
-    };
+  // Don't send the interrupted statuses to the remote client because
+  // they are not the AVRCP statuses.
+  if (status === 'mozinterruptbegin') {
+    return;
   }
 
+  var onsuccess = function() {};
```

Comment:
Same here.

`var onsuccess = function() { // To do nothing here, expect onsuccess normally. };`